### ISSUE TITLE
'h_forecast' bug when creating env without forecast for l2rpn_idf_2023

### ISSUE THREAD

**Issue by marota** [rte-france/Grid2Op#593]
## Environment
 - Grid2op version: `1.9.8`
 - System: `ubuntu20.04, ...`

## Bug description

When loading the environemnt l2rpn_idf_2023 without forecast, this error is returned:

TypeError: __init__() got an unexpected keyword argument 'h_forecast'

### Code snippet

```python
import grid2op
from grid2op.Chronics import GridStateFromFile

env = grid2op.make("l2rpn_idf_2023",
                   data_feeding_kwargs={"gridvalueClass": GridStateFromFile})
```

## Current output

```
TypeError: __init__() got an unexpected keyword argument 'h_forecast'
```

## Expected output

```
No error should be returned, and the h_forecast parameter should be disregarded in this case when not considering forecasts.
```

**Comment by BDonnot** [comment]
To deactivate the forecast you can also do :
 youhttps://grid2op.readthedocs.io/en/latest/environment.html#grid2op.Environment.BaseEnv.deactivate_forecast

(and I'll fix this ASAP)

**Comment by BDonnot** [comment]
Ideally both should be done for maximum performance. I'll work on the fix asap

**Comment by BDonnot** [comment]
Fixed in 1.10.1 :-)
